ui/react-components/components/ColorPicker/ColorPicker.module.scss

```diff
@@ -0,0 +1,46 @@
+.colorPickerWrapper {
+  position: relative;
+  display: inline-block;
+  vertical-align: middle;
+  margin: 0;
+  .colorBoxTrigger {
+    span {
```

Comment:
Also try using classnames so that they can be overridden if required

Reply:
Done